inference.py

```diff
@@ -28,11 +28,12 @@
 from scipy.io.wavfile import write
 import torch
 from mel2samp import files_to_list, MAX_WAV_VALUE
-
+from convert_model import update_model
 
 def main(mel_files, waveglow_path, sigma, output_dir, sampling_rate, is_fp16):
     mel_files = files_to_list(mel_files)
     waveglow = torch.load(waveglow_path)['model']
+    waveglow = update_model(waveglow)
```

Comment:
let's keep inference.py clean and have people converting their model manually if necessary.

Reply:
I only had manual conversion, but one of the users requested automatic, so added it.

Reply:
ok. let's not have it to be automatic and have people convert it manually. this function is not part of inference.